packages/infra/package.json

```diff
@@ -24,26 +24,26 @@
     "postupdate": "yarn"
   },
   "dependencies": {
-    "@qiwi/substrate": "^1.19.5",
-    "tslib": "^2.0.3"
+    "@qiwi/substrate": "^1.19.7",
+    "tslib": "^2.1.0"
   },
   "devDependencies": {
     "@qiwi/libdefkit": "^2.1.5",
     "@swissquote/crafty-preset-jest": "^1.13.7",
-    "@types/jest": "^26.0.15",
-    "@typescript-eslint/eslint-plugin": "^4.8.2",
-    "eslint": "^7.14.0",
-    "eslint-config-prettier": "^6.15.0",
-    "eslint-config-qiwi": "^1.10.0",
+    "@types/jest": "^26.0.20",
+    "@typescript-eslint/eslint-plugin": "^4.14.0",
+    "buildstamp": "^1.4.1",
+    "eslint": "^7.18.0",
+    "eslint-config-prettier": "^7.2.0",
+    "eslint-config-qiwi": "^1.10.3",
     "jest": "^26.6.3",
-    "prettier": "^2.2.0",
-    "prettier-config-qiwi": "^1.3.0",
+    "prettier": "^2.2.1",
+    "prettier-config-qiwi": "^1.3.2",
     "rimraf": "^3.0.2",
-    "terser": "^5.5.0",
+    "terser": "^5.5.1",
     "ts-jest": "^26.4.4",
-    "typedoc": "^0.19.2",
-    "typedoc-plugin-external-module-name": "^4.0.3",
-    "typescript": "4.0.5",
+    "typedoc": "^0.20.16",
```

Comment:
This bump requires `typedoc.json` to be updated too.

Reply:
hm, there are not any typedoc config files